test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -181,24 +185,27 @@ public void testEndToEndSnapshotAndLogEntrySync() throws Exception {
 
     }
 
-    public void testEndToEndSnapshotAndLogEntrySyncUFO(boolean diskBased, boolean checkRemainingEntriesOnSecondLogEntrySync) throws Exception {
-        testEndToEndSnapshotAndLogEntrySyncUFO(1, diskBased, checkRemainingEntriesOnSecondLogEntrySync);
+    public void testEndToEndSnapshotAndLogEntrySyncUFO(boolean diskBased, boolean checkRemainingEntriesOnSecondLogEntrySync, int numSourceClusters) throws Exception {
+        testEndToEndSnapshotAndLogEntrySyncUFO(1, diskBased, checkRemainingEntriesOnSecondLogEntrySync, numSourceClusters);
```

Comment:
can we infer this from the topology type being used?

Reply:
Same, I think it give us flexibility and is cleaner